### PR TITLE
CLI

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `README.md` modified
- `help_func` function modified
- CLI mode updated

### CODE REVIEW

File: art/__main__.py [hunk 1]
```diff
@@ -103,3 +108,7 @@ def select_test(test_name="TEST"):
             help_func()
     else:
         help_func()
```
Comment (sadrasabouri) [inline art/__main__.py:110]: As code factor mentioned, this function is too long.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: art/__main__.py (hunk 1 above)
Comment (sepandhaghighi) [reply art/__main__.py:110]: I mentioned this issue in our discord server

File: README.md [hunk 2]
```diff
@@ -734,21 +734,23 @@ _/  _   _ _/
...
-- All  :  `python -m art all yourtext`
+### CLI	
+⚠️ You can use `art` or `python -m art` to run this mode
+		
+- List of arts :  `art list` or `art arts`
+- List of fonts : `art fonts`
+- Text : `art text yourtext fontname(optional)`
+- Art : `art shape art_name` or `art art art_name`
+- Save :  `art save yourtext fontname(optional)`
+- All  :  `art all yourtext`
```
[the remark this answers is not in the document]
Comment (sepandhaghighi) [reply README.md:753]: Good point 🚀

File: art/__main__.py (hunk 1 above)
Comment (sepandhaghighi) [reply art/__main__.py:110]: fixed in https://github.com/sepandhaghighi/art/pull/220/commits/dc8bf888060bfca9911c46ed6d691f4531477837

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**